activestorage/reductionist.py

```diff
@@ -123,8 +131,16 @@ def build_request_data(source: str, bucket: str, object: str, offset: int,
         request_data["shape"] = shape
     if selection:
         request_data["selection"] = encode_selection(selection)
+    if compression:
+        # Currently the numcodecs codec IDs map to the supported compression
+        # algorithm names in the active storage server. If that changes we may
+        # need to maintain a mapping here.
```

Comment:
we need to pull out the info in this docstring somewhere where it's a bit more visible too (README doubles as docs for now, and plop a link to the table)